### PR TITLE
Cloud7

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Comment by bmwiedemann** [comment]
needs rebase

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -2702,13 +2700,13 @@ function custom_configuration()
             # set default password
             proposal_set_value provisioner default "['attributes']['provisioner']['root_password_hash']" "\"$(openssl passwd -1 $want_rootpw)\""
             # set discovery root password too
-            iscloudver 6M12plus && proposal_set_value provisioner default "['attributes']['provisioner']['discovery']['append']" "\"DISCOVERY_ROOT_PASSWORD=$want_rootpw\""
+            iscloudver 6plus && proposal_set_value provisioner default "['attributes']['provisioner']['discovery']['append']" "\"DISCOVERY_ROOT_PASSWORD=linux\""
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: -1 unintended revert here

**Comment by toabctl** [comment]
+1

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -79,7 +79,7 @@ function horizon_barclamp()
 
 function nova_role_prefix()
 {
-    if ! iscloudver 6M7plus ; then
+    if ! iscloudver 6plus ; then
```
Comment (jsuchome) [inline scripts/qa_crowbarsetup.sh]: What about ``if iscloudver 5minus`` ?

**Comment by rsalevsky** [comment]
+1

**Comment by vuntz** [comment]
+1

File: scripts/qa_crowbarsetup.sh (hunk 2 above)
Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: @jsuchome possible, didn't want to pollute this commit with further refactoring though..

**Pull request merged**